scripts/connect_clouddb_proxy.py

```diff
@@ -41,10 +47,14 @@ def main(instance_id, local_port, verbose=False):
 
     # Get the command line option
     parser = argparse.ArgumentParser(description="Connect to a google CloudSQL via a local proxy")
-    parser.add_argument('-d', '--database', default=None, required=True,
-                        help="Database, currently 'meta' or 'tess' from the config.")
-    parser.add_argument('-v', '--verbose', action='store_true', default=False,
-                        help="Print results to stdout")
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument('--from_config', default=True, action='store_true',
+                       help="Connect to all instances listed in the config file, default True.")
+    group.add_argument('--database', default=None,
```

Comment:
Now called instance in the config. Do you want to make those consistent?

Reply:
The instance is the postgres instance that holds the different databases. The name they pass here is the name for the actual database (actually a shortened version and could be anything).

We should be able to hide the `instance` details from the user and `database` is a lot more intuitive.